build/common-config.sh

```diff
@@ -37,11 +37,18 @@ var_set() {
 
 	env_val[WASM_SRC]="${WASM_SRC:-${PDO_SOURCE_ROOT}/interpreters/wasm-micro-runtime}"
 	env_desc[WASM_SRC]="
-		WASM_SRC points to the installation of the micro-wasm
+		WASM_SRC points to the installation of the wasm micro runtime
 		source in order to build the wasm interpreter
 	"
 	env_key_sort[$i]="WASM_SRC"; i=$i+1; export WASM_SRC=${env_val[WASM_SRC]};
 
+        env_val[WASM_MODE]="${WASM_MODE:-INTERP}"
```

Comment:
superfluous spaces?

Reply:
Ah yes, this looks like an extra space. I know it's aesthetic but will definitely fix.

Reply:
As an aside: Something odd is happening with how my emacs is rendering this spacing because it looked aligned in my emacs, but git was certainly adding in an extra space there.